pkgs/applications/video/anime-downloader/default.nix

```diff
@@ -31,6 +33,10 @@ python3.pkgs.buildPythonApplication rec {
     tabulate
   ]);
 
+  preFixup = ''
+    wrapQtApp "$out/bin/anime" --prefix PATH : ${lib.makeBinPath propagatedBuildInputs}
+  '';
```

Comment:
Is this double wrapping and why are we wrapping propagatedBuildInputs? That should work automatically.

Reply:
Without `preFixup` I got:

```sh
$ ./result/bin/anime gui
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted (core dumped)
```

That's also why this was packaged with `doCheck = false;`, QT environment doesn't seem to function without those "workarounds".

Reply:
double wrapping is only a problem on darwin and this package is restricted to linux